eZ/Publish/Core/Persistence/Legacy/Content/Location/Mapper.php

```diff
@@ -45,6 +46,9 @@ public function createLocationFromRow(array $data, $prefix = '', Location $locat
         $location->depth = (int)$data[$prefix . 'depth'];
         $location->sortField = (int)$data[$prefix . 'sort_field'];
         $location->sortOrder = (int)$data[$prefix . 'sort_order'];
+        if (isset($data[$prefix . 'trashed'])) {
+            $location->trashed = new DateTime((int)$data[$prefix . 'trashed']);
```

Comment:
From what I can see, this should be timestamp, the DateTime  conversion should be done in:
https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/Repository/TrashService.php#L328-L345

Which is missing mapping of trashed currently, meaning api value has a null value atm.